CONTRIBUTING.md

```diff
@@ -90,8 +90,8 @@ For command-line examples and documentation, we follow the [docopt usage pattern
 - mutually exclusive elements are given between `(` and `)` and separated by `|`.
 
 ```diff
-- npm start [ $service_id ] [, $service_id, ...]
-+ npm start [service_id]...
+- npm start --services [ $service_id ] [, $service_id, ...]
++ npm start --services [service_id...]
```

Comment:
This would not work. We would need:
```suggestion
- npm start -- --services [ $service_id ] [, $service_id, ...]
+ npm start -- --services [service_id...]
```

For the purposes of illustrating docopt, I am not sure this change is very relevant, I think it makes things more complicated. I suggest a rollback:
```suggestion
- npm start [ $service_id ] [, $service_id, ...]
+ npm start [service_id...]
```